test/broadcast.jl

```diff
@@ -990,3 +990,18 @@ p0 = copy(p)
     @test isequal(identity.(Vector{<:Union{Int, Missing}}[[1, 2],[missing, 1]]),
                   [[1, 2],[missing, 1]])
 end
+
+@testset "Issue #28382: eltype inconsistent with getindex" begin
+    struct Cyclotomic <: Number
+    end
+
+    Base.getindex(x::Cyclotomic, i::Integer) = i
+
+    Base.length(x::Cyclotomic) = 1
+    Base.eltype(::Type{<:Cyclotomic}) = Tuple{Int,Int}
+
+    Base.:*(c::T, x::Cyclotomic) where {T<:Real} = [1, 2]
+    Base.:*(x::Cyclotomic, c::T) where {T<:Real} = [1, 2]
+
+    @test Cyclotomic() .* [2, 3] == [[1, 2], [1, 2]]
+end
```

Comment:
```suggestion
end

```
Looks like a missing trailing newline.
@staticfloat Shouldn't this have been caught by the whitespace buildbot?

Reply:
No, I think the whitespace buildbot only checks for trailing whitespace.

Reply:
Ah, I see, it's marked as a TODO